### PR TITLE
Exposes gene meta description

### PR DESCRIPTION
**Pull request by dzucconi**
Along with some related clean up. Notably I'm altering `markdown(format: PLAIN)` to return actual plaintext (given a markdown field, stripping markdown related syntax).

The meta#description logic follows from the Force model: 
https://github.com/artsy/force/blob/5ee1ca79ecb9f086932d4148b3a8d8c19b724e1a/src/desktop/models/gene.coffee#L37-L43

### CODE REVIEW

File: src/schema/v2/sorts/fair_sorts.ts [hunk 1]
```diff
@@ -16,6 +16,6 @@ export const FairSorts = {
   }),
 }
 
-export type TFairSorts = keyof typeof FAIR_SORTS
+export type TFairSort = typeof FAIR_SORTS[keyof typeof FAIR_SORTS]["value"]
```
Comment (damassi) [inline src/schema/v2/sorts/fair_sorts.ts:19]: (thought) Re `TFairSort` -- Know this code was here before, but wanted to call out that prefixing our types with `T` or interfaces with `I` is unconventional and, at least on our FE codebases, has been explicitly avoided in the past. 

(I'm fine with moving in this direction, but we should be deliberate about this stuff and try to stick to idiom unless that idiom has a legitimate technical cost -- in which case, we should fix the problem.)

Comment (dzucconi) [reply src/schema/v2/sorts/fair_sorts.ts:19]: I don't really like it either TBH — the problem I've been having lately in Metaphysics is just avoiding naming the type the same thing as the enum or type. Typically I'd name the enum object `FairSortEnum` and have the type be `FairSort`. I can give this another look though.

If we're being honest: what is conventional in Metaphysics at this point? — it's a free-for-all of disorganized code. Conventions would be lovely.

Comment (damassi) [reply src/schema/v2/sorts/fair_sorts.ts:19]: Sometimes conventions can be defined by what a thing is not. (And tha of course can be problematic, but not necessarily so.) I hear ya tho about MP. Our TypeScript layer is still pretty new, so there's room to improve and to set some conventions down if necessary, or to correct bad ones should they be observed.

File: src/schema/v2/fairs.ts [hunk 2]
```diff
@@ -88,7 +88,20 @@ const Fairs: GraphQLFieldConfig<void, ResolverContext> = {
 
 export default Fairs
 
-export const fairsConnection = {
+export const fairsConnection: GraphQLFieldConfig<
```
Comment (mzikherman) [inline src/schema/v2/fairs.ts:91]: nice types!

File: src/schema/v2/gene.ts [hunk 3]
```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
 
 const SUBJECT_MATTER_REGEX = new RegExp(SUBJECT_MATTER_MATCHES.join("|"), "i")
 
+const META_DESCRIPTION_OVERRIDES = {
+  "western-europe":
+    "Discover Western European artists from pre-history to present, and browse works by size, prize and medium.",
```
Comment (mzikherman) [inline src/schema/v2/gene.ts:42]: typo **price**, and medium

File: src/schema/v2/gene.ts [hunk 4]
```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
...
+  "latin-america-and-the-caribbean":
+    "Discover artists from Latin America and the Caribbean from pre-history to present, and browse works by size, prize and medium.",
```
Comment (mzikherman) [inline src/schema/v2/gene.ts:44]: same typo, **price**

File: src/schema/v2/gene.ts [hunk 5]
```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
...
+
 export const GeneType = new GraphQLObjectType<any, ResolverContext>({
   name: "Gene",
   interfaces: [NodeInterface, Searchable],
   fields: () => {
     const { filterArtworksConnection } = require("./filterArtworksConnection")
     return {
+      meta: {
+        resolve: (gene) => gene,
+        type: new GraphQLNonNull(
```
Comment (mzikherman) [inline src/schema/v2/gene.ts:62]: nice, this is non-null!

File: src/schema/v2/gene.ts [hunk 6]
```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
...
+          new GraphQLObjectType<any, ResolverContext>({
+            name: "GeneMeta",
+            description: "Meta tags for Gene pages",
+            fields: {
+              description: {
+                type: new GraphQLNonNull(GraphQLString),
```
Comment (mzikherman) [inline src/schema/v2/gene.ts:68]: also, non-null!

File: src/schema/v2/gene.ts [hunk 7]
```diff
@@ -33,12 +37,59 @@ const SUBJECT_MATTER_MATCHES = [
...
+                  if (META_DESCRIPTION_OVERRIDES[gene.id] !== undefined) {
+                    return META_DESCRIPTION_OVERRIDES[gene.id]
+                  }
+
+                  if (
+                    gene.type?.name?.includes(
+                      "Geographical Regions and Countries"
+                    )
+                  ) {
+                    return `Explore art by artists who are from, or who have lived in, ${gene.name}. Browse works by size, price, and medium.`
```
Comment (mzikherman) [inline src/schema/v2/gene.ts:79]: tricksy

**Review by mzikherman — APPROVED** [review]
LGTM! couply minor typos

File: src/schema/v2/gene.ts (hunk 3 above)
Comment (dzucconi) [reply src/schema/v2/gene.ts:42]: Ah, weird; these were copied out of Force :p

**Comment by dzucconi** [comment]
Forgot about this one; going to rebase and get it merged

**Pull request merged**